src/bidmanager.js

```diff
@@ -43,6 +43,7 @@ function getBidders(bid) {
 function bidsBackAdUnit(adUnitCode) {
   const requested = $$PREBID_GLOBAL$$._bidsRequested
     .map(request => request.bids
+      .filter(adUnitsFilter.bind(this, $$PREBID_GLOBAL$$._adUnitCodes))
```

Comment:
It's isn't needed here, only in places where the filter may be passed in rather than the current `pbjs._adUnitCodes` array. I'll remove from here and elsewhere where we can and have `utils.adUnitsFilter` check for an array passed in and if not found use ad unit codes defined on global.